core/src/main/scala/com/evolutiongaming/kafka/journal/HostName.scala

```diff
@@ -12,10 +12,68 @@ object HostName {
   def of[F[_]: Sync](): F[Option[HostName]] = {
     Sync[F].delay {
       for {
-        a <- com.evolutiongaming.hostname.HostName()
+        a <- EvoHostName()
       } yield {
         HostName(a)
       }
     }
   }
 }
+
+import java.net.InetAddress
+import java.util.concurrent.Executors
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.sys.process.*
+import scala.util.Properties
+import scala.util.control.NonFatal
+
+/** Copied from https://github.com/evolution-gaming/hostname */
```

Comment:
is this temporary copy till `hostname` gets Scala 3 artifacts?

Reply:
Both yes and no. 
Some of evo single-class libraries make almost no sense to me, so I decided to put this one here, especially since there's a wrapper for it anyway. 
Also I was not able to build the `hostname` easily, there are some weird problems with JVM and SBT or something like that.

If you feel like it should stay as a dependency, I'll update the library. But if you agree on it making no sense, then I'll refactor this file and make it nice.